docs/tutorials/load-test-http-multiple.md

```diff
@@ -0,0 +1,70 @@
+---
+template: main.html
+---
+
+# Load Test Multiple HTTP endpoints
+
+[Your first experiment](../getting-started/your-first-experiment.md) describes how to load test a single endpoint from an HTTP service inside Kubernetes. This tutorial expands on the previous tutorial and describes how to load test multiple endpoints from an HTTP service.
+
+<p align='center'>
+  <img alt-text="load-test-http" src="../../getting-started/images/http.png" />
+</p>
+
+???+ warning "Before you begin"
+    1. Ensure that you have a Kubernetes cluster and the [`kubectl` CLI](https://kubernetes.io/docs/reference/kubectl/). You can create a local Kubernetes cluster using tools like [Kind](https://kind.sigs.k8s.io/) or [Minikube](https://minikube.sigs.k8s.io/docs/).
+    2. Deploy the sample HTTP service in the Kubernetes cluster.
+    ```shell
+    kubectl create deploy httpbin --image=kennethreitz/httpbin --port=80
+    kubectl expose deploy httpbin --port=80
+    ```
+
+***
+
+## Launch experiment
+Launch the Iter8 experiment inside the Kubernetes cluster.
+
+```bash
+iter8 k launch \
+--set "tasks={ready,http,assess}" \
+--set ready.deploy=httpbin \
+--set ready.service=httpbin \
+--set ready.timeout=60s \
+--set http.endpoints.get.url=http://httpbin.default/get \
```

Comment:
I think it would be useful to have a parameter set at the base level such as `--set http.numRequests=200` that applies to all. This is different from the `qps` example which relies on the default value.